tools/travis/stage.sh

```diff
@@ -81,12 +81,12 @@ flink-formats/flink-parquet,\
 flink-formats/flink-sequence-file,\
 flink-formats/flink-json,\
 flink-formats/flink-csv,\
+flink-formats/flink-orc,\
```

Comment:
Hi @zentol , Can you help me see if it's right here? Is this the only one that needs to be changed?